.github/workflows/pull-request.yml

```
@@ -378,6 +378,8 @@ jobs:
             kill "$(cat .pidfile_prod)"
           fi
       - name: Start Development App
+        run: |
+          npm run serve &
         run: |
```

actionlint (1.7.7)

key "run" is duplicated in element of "steps" section. previously defined at line:381,col:9

(syntax-check)